recipe/meta.yaml

```diff
@@ -8,10 +8,10 @@ package:
 source:
   fn:  {{ filename }}
   url: http://sourceforge.net/projects/boost/files/boost/{{ version }}/{{ filename }}
-  md5: 6095876341956f65f9d35939ccea1a9f
+  md5: 5fb94629535c19e48703bdb2b2e9490f
```

Comment:
Do they provide a better checksum them md5? Really any sha would be great. Preferably sha256.